doc/developer-guides/hld/virtio-net.rst

```diff
@@ -533,7 +533,7 @@ a PCI slot to the device model acrn-dm as shown below.
 
 .. code-block:: none
 
-   -s 4,virtio-net,<macvtap_name>,[mac=<XX:XX:XX:XX:XX:XX>]
+   -s 4,virtio-net,macvtap=<name>,[mac=<XX:XX:XX:XX:XX:XX>]
```

Comment:
I thought the only valid options here were `tap` and `vmnet`, have I missed something?

Reply:
As we have talk, we should remove the macvtap from doc.